Cargo.toml

```diff
@@ -38,16 +39,24 @@ js-sys = { version = "0.3", optional = true }
 leb128 = "0.2"
 pyo3 = { version = "0.17", features = ["extension-module", "abi3", "abi3-py37", "generate-import-lib"], optional = true }
 rand = "0.8"
+reqwest = { version = "0.11.14", features = ["rustls-tls"], default-features = false }
```

Comment:
`cargo add` use this format, I'll keep following it and not manually change it. Cargo will use `0.11.15` if available since it follows semver anyway. The only difference with `0.11`: it will just refuse `0.11.13`

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Reply:
Actually specifying all version numbers blocks minor upgrades *only* for crates in `0.x.x` which are still unstable, which is a good thing. For crates in `X.x.x` with `X>0` specifying all version numbers *allows* minor upgrades.

Therefore, I think we can start specifying dependencies using all version numbers in all our crates.

Reply:
It will be moved to CloudproofRust later on but could you please make this dependency optional?

Reply:
Done in 4155642